tailwind.config.js

```diff
@@ -6,7 +6,46 @@ module.exports = {
         lightblue: '#18A0FB', // an example of extending tailwind
       },
     },
+    borderRadius: {
+      sm: '10px',
+    },
+    width:{
+      '1/5':'257px'
+    },
+    height:{
+      '3/12':'290px'
+    }
   },
   variants: {},
-  plugins: [],
+  plugins: [
+    function ({ addComponents }) {
```

Comment:
Why is this?

Reply:
Because I needed customized style that I could not implemented with Tailwindcss classes.